### PR TITLE
Allow configuring if existing headers should be skipped in license check

### PR DESCRIPTION
**Pull request by hashhar**
All credit to @findinpath for finding this.

See for source https://github.com/trinodb/trino/pull/11563#issuecomment-1072302541

cc: @findinpath @findepi

### CODE REVIEW

File: airbase/pom.xml [hunk 1]
```diff
@@ -738,7 +739,7 @@
                     </dependencies>
                     <configuration>
                         <skip>${air.check.skip-license}</skip>
-                        <skipExistingHeaders>true</skipExistingHeaders>
+                        <skipExistingHeaders>${air.license.skip-existing-headers}</skipExistingHeaders>
```
Comment (findepi) [inline airbase/pom.xml:742]: Why not always `false`?

Comment (findepi) [reply airbase/pom.xml:742]: (it's `true` since ever (d71d25e711593aff2a1e339d146d8a83205e2996))

Comment (hashhar) [reply airbase/pom.xml:742]: Just to preserve compatibility. IMO it should be false.

Comment (findepi) [reply airbase/pom.xml:742]: @electrum can you please chime in here?

Comment (electrum) [reply airbase/pom.xml:742]: Let’s make this false, since that is the better default. Any projects that need to skip can set the property when updating.

Comment (hashhar) [reply airbase/pom.xml:742]: Updated default to true and adjusted CHANGES entry accordingly.

**Review by findepi — APPROVED** [review]

**Pull request merged**